src/ontology/uberon.Makefile

```diff
@@ -1025,147 +1095,144 @@ subsets/%-view.owl: uberon.owl contexts/context-%.owl tmp/simple-slim-seed.txt
 		--set-ontology-id $(URIBASE)/uberon/$@ -o -f ttl $@
 
 
-
 # ----------------------------------------
-# COMPOSITE STAGES
+# COMPOSITE ANATOMY
 # ----------------------------------------
 
-$(TMPDIR)/update-stages: $(SRC) | $(TMPDIR)
-	rm -rf $(TMPDIR)/developmental-stage-ontologies && \
-	cd $(TMPDIR) && \
-	git clone https://github.com/obophenotype/developmental-stage-ontologies.git
-
-$(TMPDIR)/developmental-stage-ontologies/src/ssso-merged.obo: $(TMPDIR)/update-stages
-	test -f $@
-
-$(TMPDIR)/merged-stages-xrefs.obo: $(TMPDIR)/developmental-stage-ontologies/src/ssso-merged.obo
-	$(ROBOT) query -i $(TMPDIR)/developmental-stage-ontologies/src/ssso-merged.obo \
-		       --query ../sparql/xrefs-to-uberon.sparql $@_xrefs_to_uberon.tsv
-	cat $@_xrefs_to_uberon.tsv | tail -n +2 | \
-		$(SCRIPTSDIR)/tbl2obolinks.pl -k  --rel xref - > $@.tmp && mv $@.tmp $@
-
-$(TMPDIR)/composite-stages.obo: $(TMPDIR)/merged-stages-xrefs.obo
-	$(ROBOT) merge -i tmp/developmental-stage-ontologies/src/ssso-merged-uberon.obo \
-		       -i $(TMPDIR)/merged-stages-xrefs.obo \
-		 convert -f obo --check false -o $@
-
-
+# Source files for the collected/composite ontologies
 # ----------------------------------------
-# COMPOSITE ANATOMY
+COLLECTED_drosophila_SOURCES =       $(IMPORTDIR)/local-fbbt.owl \
+				     $(IMPORTDIR)/local-fbdv.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-fbbt.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-fbdv.owl \
+				     $(BRIDGEDIR)/cl-bridge-to-fbbt.owl
+
+COLLECTED_worm_SOURCES =             $(IMPORTDIR)/local-wbbt.owl \
+				     $(IMPORTDIR)/local-wbls.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-wbbt.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-wbls.owl \
+				     $(BRIDGEDIR)/cl-bridge-to-wbbt.owl
+
+COLLECTED_zebrafish_SOURCES =        $(IMPORTDIR)/local-zfa.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-zfa.owl \
+				     $(BRIDGEDIR)/cl-bridge-to-zfa.owl
```

Comment:
I am noticing here that there is no specific zfs related import, which is probably fine, but I wanted it said; For now, zfa comes with zfs built in, but this may change in the next year.

Reply:
Overall, and for now, the composition of the various collected/composite products has been left mostly unchanged. I did not include ZFS here because the original `composite-zebrafish` did not include it.

If/when we need to explicitly include ZFS, it would be very easy to do so. Just add a `$(IMPORTDIR)/local-zfs.owl` import and update this variable to include that file as well as the (existing but currently unused) `$(BRIDGEDIR)/uberon-bridge-to-zfs`.

Reply:
I just wanted you to be aware, makes sense!